.github/workflows/build.yml

```diff
@@ -65,9 +65,12 @@ jobs:
     - name: Run SCSS Linter
       run:  docker-compose exec -T web /bin/sh -c 'bundle exec rake lint:scss'
 
-    - name: Run tests
+    - name: Run Rspec tests
       run: docker-compose exec --env COVERAGE=true -T web /bin/sh -c 'bundle exec rake spec'
 
+    - name: Run Javascript tests
+      run: docker-compose exec --env -T web /bin/sh -c 'yarn run tests'
```

Comment:
Does this need to be `yarn run test`?

Reply:
yup good spot fixed in https://github.com/DFE-Digital/register-trainee-teachers/pull/589/commits/d0369e3d70b1b5c41ce280554dfbbe267c1e4125